api/v1/controllers/bots.js

```diff
@@ -58,7 +59,44 @@ module.exports = {
    * @param {Function} next - The next middleware function in the stack
    */
   getBot(req, res, next) {
-    doGet(req, res, next, helper);
+    const resultObj = { reqStartTime: req.timestamp };
+
+    // try to get cached entry
+    redisCache.get(req.swagger.params.key.value, (cacheErr, reply) => {
```

Comment:
This appears to be trying to use the bot name or id as the cache key but we need to stop doing that (there's a recent story for that) because it risks name collisions where someone asked for a lens named "foo" but gets a bot named "foo" instead!
Use "/v1/bots/[NAME_OR_ID]" as the cache key, and lenses and perspectives and subjects and aspects etc. will all be doing the same soon.